cogboard-webapp/src/components/widgets/types/LogViewerWidget/LogList/styled.js

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled/macro';
+import { COLORS } from '../../../../../constants';
+import { Typography, Accordion } from '@material-ui/core';
+
+export const Container = styled.div`
+  max-height: 100%;
+  display: grid;
+  padding-top: 76px;
```

Comment:
Please verify if other components also use pixels. Maybe try with relative units (em, rem)

Reply:
Check it also for all the components below